README.md

```diff
@@ -244,7 +211,7 @@ new Promise((resolve, reject) => {
 }) -> fresh undelegated promise
 
 
-Promise.delegate((resolve, reject, resolveWithPresence) => {
+new Promise.delegated((resolve, reject, resolveWithPresence) => {
```

Comment:
```suggestion
Promise.delegated((resolve, reject, resolveWithPresence) => {
```
I expect `Promise.delegated` to be a non-constructor function. If we do make it a constructor, it should be spelled `Promise.Delegated`.

Making it a function fits with our `Proxy` analogy. `Proxy` is a constructor. `Proxy.revocable` is a non-constructor function.